### PR TITLE
PA-841 : Automation of the MySQL workload bench.

### PR DESCRIPTION
**Pull request by jkagliwal-px**
Created a Docker image which performs the following operations on mysql server- 

1. Creates a custom database.
2. Performs CRUD on the database and fill the storage untill the pod is manually killed/ testcase exits.

PS: This is a NON-TPCC workload requirement. 

Jenkins logs : https://jenkins.pwx.dev.purestorage.com/job/PDS/job/pds-branch-build-test/268/console

### CODE REVIEW

File: drivers/pds/lib/utils.go [hunk 1]
```diff
@@ -190,6 +190,7 @@ const (
 	pdsTpccImage                 = "portworx/torpedo-tpcc-automation:v1"
 	redisStressImage             = "redis:latest"
 	rmqStressImage               = "pivotalrabbitmq/perf-test:latest"
+	mysqlBenchImage              = "pwxbuild/pds-mysqlbench:v1"
```
Comment (dbhatnagar-px) [inline drivers/pds/lib/utils.go:193]: Please push this to portworx repo

File: drivers/pds/lib/utils.go [hunk 2]
```diff
@@ -1849,6 +1850,64 @@ func CreateRedisWorkload(name string, image string, dnsEndpoint string, pdsPassw
...
+	value = append(value, "pds")
+	value = append(value, pdsPassword)
+
+	for index := range env {
+		deploymentSpec.Spec.Template.Spec.Containers[0].Env[index].Name = env[index]
+		deploymentSpec.Spec.Template.Spec.Containers[0].Env[index].Value = value[index]
+	}
+	deployment, err := k8sApps.CreateDeployment(deploymentSpec, metav1.CreateOptions{})
+	if err != nil {
+		log.Errorf("An Error Occured while creating deployment %v", err)
```
Comment (dbhatnagar-px) [inline drivers/pds/lib/utils.go:1896]: Please remove log.Errorf

File: drivers/pds/lib/utils.go [hunk 3]
```diff
@@ -1849,6 +1850,64 @@ func CreateRedisWorkload(name string, image string, dnsEndpoint string, pdsPassw
...
+		return nil, err
+	}
+	err = k8sApps.ValidateDeployment(deployment, timeOut, timeInterval)
+	if err != nil {
+		log.Errorf("An Error Occured while validating the pod %v", err)
```
Comment (dbhatnagar-px) [inline drivers/pds/lib/utils.go:1901]: Please remove log.Errorf

File: drivers/pds/lib/utils.go (hunk 2 above)
Comment (jkagliwal-px) [reply drivers/pds/lib/utils.go:1896]: Done!

File: drivers/pds/lib/utils.go (hunk 3 above)
Comment (jkagliwal-px) [reply drivers/pds/lib/utils.go:1901]: Done!

File: tests/pds/common.go [hunk 4]
```diff
@@ -151,6 +152,11 @@ func RunWorkloads(params pdslib.WorkloadGenerationParams, ds PDSDataService, dep
 		log.Infof("Running Workloads on DataService %v ", ds.Name)
 		pod, dep, err = pdslib.CreateDataServiceWorkloads(params)
 	}
+	if ds.Name == mysql {
+		params.DeploymentName = *deployment.ClusterResourceName
+		log.Infof("Running Workloads on DataService %v ", ds.Name)
```
Comment (Madanagopal19) [inline tests/pds/common.go:157]: use InfoD

**Review by Madanagopal19 — APPROVED** [review]
minor comment

**Review by aravindranath14 — APPROVED** [review]

**Review by dbhatnagar-px — APPROVED** [review]
lgtm

**Pull request merged**